### DISCUSSION TITLE
Unexpected identifier in x-data

### DISCUSSION THREAD

**Discussion by RicardoRamirezR** [alpinejs/alpine#3406] [1. Help] [answered]
Hi there,

Given the following code:
```php
<div x-data="{ id: 35020, name: '{{ $name }}' }">
```
All works well if ```$name``` does not contains a single quotation mark or double quotation mark. The problem arises when ```$name = "T'chala"``` or ```$name = 'T"chala'```, I got ```Uncaught SyntaxError: Unexpected identifier 'chala'```

Any idea will be much appreciated

**Comment by ekwoka** [comment, marked as answer]
use backticks instead, or if your templating engine has it, an `escape` filter

**Reply by RicardoRamirezR** [reply]
@ekwoka thank you! I've used backticks
